src/components/course/list/common/CourseTypeFilter.vue

```diff
@@ -39,9 +37,21 @@
                 emit("update:selectedType", type);
             }
 
+            function shortNameFor(type: CourseType) {
+                switch (type) {
+                    case CourseType.PG: {
+                        return "PG";
+                    }
+                    default: {
+                        return type;
+                    }
+                }
+            }
+
```

Comment:
Do you always need a short form? As I explained yesterday, CourseTypes will soon be a configuration by the backend, so we would not know the values on compile time. We could maybe propose pairs of type + short form

Reply:
I think that would be nice to always have a short-form for the mobile UI.